doc/texture_packs.txt

```diff
@@ -72,6 +72,11 @@ by texture packs. All existing fallback textures can be found in the directory
       `crosshair_color` and `crosshair_alpha` are used to create a cross
       when no texture was found
 
+* `object_crosshair.png`
+* the crosshair seen when pointing at an object. The settings
+`object_crosshair_color` and `object_crosshair_alpha` are used to create a cross
+when no texture was found
```

Comment:
These lines probably require indentation. Looks like 4 spaces, going by L71.

Reply:
How is it now?

Reply:
It looks about right. :+1: